packages-web/maps/src/utils/Validations.ts

```diff
@@ -54,6 +54,14 @@ export const validateLocationProps = <T extends Partial<Container.MapsContainerP
                             location.staticLongitude
                         }' at location ${index + 1}`
                     );
+                } else {
+                    if (location.onClickEvent && location.onClickEvent !== "doNothing") {
+                        errorMessage.push(
```

Comment:
In error feedback and validation, use the terminology used in the property configuration
``` ts
 errorMessage.push(`Static location items (at location ${index + 1}) does not support maker events. The On click can only be set to Do nothing`);
```

Reply:
Done